pkgs/top-level/python-packages.nix

```diff
@@ -12747,840 +12600,162 @@ in {
     };
   };
 
-  oslosphinx = buildPythonPackage rec {
-    name = "oslosphinx-3.3.1";
+  rfc3986 = callPackage ../development/python-modules/rfc3986 { };
+
+   cachetools_1 = callPackage ../development/python-modules/cachetools/1.nix {};
```

Comment:
indentation?

Reply:
weird, i fix this commit.

Reply:
Actually, I think at some point the indentation is also incorrect in `python-packages.nix` already before these changes.

Reply:
Yes, i saw weird indentation in the `python-packages.nix` but i haven't touched it to keep the PR clean.